packages/shared/src/components/cards/squad/SquadsDirectoryFeed.tsx

```diff
@@ -7,21 +7,27 @@ import {
 import HorizontalScroll from '../../HorizontalScroll/HorizontalScroll';
 import { UnfeaturedSquadGrid } from './UnfeaturedSquadGrid';
 import { SquadGrid } from './SquadGrid';
+import { useViewSize, ViewSize } from '../../../hooks';
+import { SquadList } from './SquadList';
+import { Button, ButtonVariant } from '../../buttons/Button';
 
 interface SquadHorizontalListProps {
   title: ReactNode;
   query: SourcesQueryProps;
   linkToSeeAll: string;
   className?: string;
+  children?: ReactNode;
 }
 
-export function SquadHorizontalList({
```

Comment:
Renamed as HorizontalScroll is now too specific and not applicable on Mobile view.